TIL/topics/java_basic/2025.06.17 .md

```
@@ -0,0 +1,78 @@
+ - build: 빌드 파일 수정
+ - ci: CI 설정 파일 수정
+ - perf: 성능 개선
+ - chore: 빌드 업무 수정, 패키지 매니저 수정
+ - rename: 파일 혹은 풀더 명을 수정만 한 경우
+ - remove: 파일을 삭제만 한 경우
+
+
+## 객체 지향의 3대 원칙 (4대의 경우 추상화 포함)
+ - 캡슐화 (encapsulation)
+ - 상속
+ - 다양성
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)